Scripts/Editor/NodeEditorGUI.cs

```diff
@@ -117,39 +117,80 @@ public partial class NodeEditorWindow {
         }
 
         /// <summary> Draw a bezier from output to input in grid coordinates </summary>
-        public void DrawNoodle(Color col, List<Vector2> gridPoints) {
+        public void DrawNoodle(Gradient grad, List<Vector2> gridPoints, bool disableHover = false) {

```

Comment:
Use full name `gradient` here as well. I know the previous was `col` but `grad` just sounds odd.

Reply:
What does `disableHover` do? Also please invert this bool so that that it isn't *disableX*